From: Svante Signell <svante.signell@telia.com>
Date: Sun, 1 Jan 2012 18:42:05 +0100
Subject: hurd fix

Solves failure to build from sources on Debian GNU/Hurd,
see http://bugs.debian.org/641283 for details.
---
 vobcopy.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/vobcopy.h b/vobcopy.h
index ebe2709..19b40e1 100644
--- a/vobcopy.h
+++ b/vobcopy.h
@@ -69,7 +69,7 @@ typedef enum  { FALSE=0, TRUE=1 }  bool;
 /* //////////  *BSD //////////  */
 #if ( defined( BSD ) && ( BSD >= 199306 ) )
 
-#if !defined( __NetBSD__ ) || \
+#if !defined( __NetBSD__ ) && !defined(__GNU__) || \
        ( defined( __NetBSD__) && ( __NetBSD_Version__ < 200040000 ) )
 #include <sys/mount.h>
 #define USE_STATFS 1
